tests/api.dylan

```diff
@@ -109,12 +108,12 @@ end function-test regex-group-count;
 define regular-expressions function-test regex-search ()
   // Test case-sensitive parameter
   // See bug 7371
+  check-false("regex-search(..., case-sensitive: #t) works on character sets",
+             regex-search(compile-regex("[a-z]"), "A", case-sensitive: #t));
   check-true("regex-search(..., case-sensitive: #f) works for character sets",
              regex-search(compile-regex("[a-z]"), "A", case-sensitive: #f));
-  check-true("regex-search(..., case-sensitive: #t) works on character sets",
-             regex-search(compile-regex("[a-z]"), "A", case-sensitive: #f));
   check-false("case-sensitive: #t works for regular strings",
-              regex-search(compile-regex("abc"), "aBc", case-sensitive: #t));
+             regex-search(compile-regex("abc"), "aBc", case-sensitive: #t));
```

Comment:
The indentation is slightly off.  Make sure the regex-search on the continuation line start directly under the double quote on the previous line.  Could be a Tab issue?  Easiest way to get this right is to use dylan-mode in Emacs, but that's not much help if you're not an Emacs fan.

Could you remove "See bug 7371" since that refers to an old Bugzilla bug that no longer exists?  Thanks.

Reply:
I am using Emacs+dylan-mode, I've now set a hook to make sure `indent-tabs-mode` is nil.